test/services/beanstalk/deployable-artifact-test.js

```diff
@@ -56,7 +56,7 @@ describe('deployable artifact module', function () {
             let deleteEbextensionsStub = sandbox.stub(ebextensions, 'deleteAddedEbExtensionsFromDirectory').returns(true);
             let copyDirectoryStub = sandbox.stub(util, 'copyDirectory').returns(Promise.resolve(true));
 
-            return deployableArtifact.prepareAndUploadDeployableArtifact(serviceContext, [])
+            return deployableArtifact.prepareAndUploadDeployableArtifact(accountConfig,serviceContext, [])
```

Comment:
Once you remove the accountConfig parameters in these methods you can remove their references here.